### PR TITLE
Feature/issue429

### LINKED ISSUE #429: Edit Customer groups topic to include steps to create customer groups for discounts
There is no steps to create a 

Customer groups related to a specific discounts available.

Regarding: https://docs.magento.com/m2/ee/user_guide/customers/customer-groups.html

### PR DESCRIPTION
**Pull request by carolinabett**
Fixes #429 

## Purpose of this pull request

Update the Customer groups topic with instructions to associate customer groups with specific discounts.

## Affected documentation pages

https://docs.magento.com/m2/ee/user_guide/customers/customer-groups.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [ ] B2B

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
Awesome @carolinabett! Thanks for the PR!!!

File: src/customers/customer-groups.md [hunk 1]
```diff
@@ -74,6 +74,19 @@ _Customer Groups_
       ![]({% link images/images/stores-customer-group-assign.png %}){: .zoom}
       _Assign a Customer Group_
 
+## To relate a group of customers to specific discounts:
```
Comment (jeff-matthews) [inline src/customers/customer-groups.md:77]:
```suggestion
## To relate a group of customers to specific discounts
```

File: src/customers/customer-groups.md [hunk 2]
```diff
@@ -74,6 +74,19 @@ _Customer Groups_
...
+
+1. On the _Admin_ sidebar, go to **Marketing** > **Cart Price Rules**.
+
+1. Create a New Rule or Edit an existing rule.
```
Comment (jeff-matthews) [inline src/customers/customer-groups.md:81]:
```suggestion
1. Create a new rule or edit an existing rule.
```

File: src/customers/customer-groups.md [hunk 3]
```diff
@@ -74,6 +74,19 @@ _Customer Groups_
...
+
+1. Set which customer groups the rule will be used to.
```
Comment (jeff-matthews) [inline src/customers/customer-groups.md:83]:
```suggestion
1. Select the customer groups the rule will apply to.
```

File: src/customers/customer-groups.md [hunk 4]
```diff
@@ -74,6 +74,19 @@ _Customer Groups_
...
+
+    ![]({% link images/images/stores-customer-group-discounts.png %}){: .zoom}
+      _Customer Group to Specific Discounts_
+
+1. When complete, click <span class="btn">Save</span>.
```
Comment (jeff-matthews) [inline src/customers/customer-groups.md:88]:
```suggestion
1. Click <span class="btn">Save</span>.
```

File: src/customers/customer-groups.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (carolinabett) [reply src/customers/customer-groups.md]: Hello @meker12 , I sent a new commit with the corrections as your suggestion. Thank you!

File: src/customers/customer-groups.md [hunk 5]
```diff
@@ -74,6 +74,21 @@ _Customer Groups_
...
+1. Select the cart price rule with the discount to apply, or [create a new price rule]({% link marketing/price-rules-catalog.md %}).
+
+1. Select the customer groups the rule will apply to.
+
+    ![]({% link images/images/stores-customer-group-discounts.png %}){: .zoom}
+      _Customer Group to Specific Discounts_
+
+1. Click <span class="btn">Save</span>.
+
+_**Note:**_ You can also use Advance pricing to apply product discounts to customer groups. See [Advanced pricing]({% link catalog/product-price-group.md %}).
```
Comment (jeff-matthews) [inline src/customers/customer-groups.md:90]:
```suggestion
{:.bs-callout-info}
You can also use Advance pricing to apply product discounts to customer groups. See [Advanced pricing]({% link catalog/product-price-group.md %}).
```

**Pull request merged**